frontend/app/instance-initializers/x-push-state.js

```diff
@@ -1,9 +1,9 @@
 import Ember from 'ember';
 
-var XContentReady = new CustomEvent('XContentReady');
+let XContentReady = new CustomEvent('XContentReady');
```

Comment:
We can probably remove XPushState, it's horribly buggy and we have something better now: FastBoot™!